src/app/layout/components/Sidebar.tsx

```diff
@@ -10,20 +10,20 @@ interface Props {
 }
 
 export const Sidebar = ({ isSidebarOpen, toggleSidebar }: Props) => {
-  const sideBarRoutes = SideBarRoutes()
+  const sideBarRoutes = SideBarRoutes();
```

Comment:
What's `SidebarRoutes`?

Reply:
I have change this  `SidebarRoutes` by `roleBasedRoutes` because here we get routes on the base of User Role

Reply:
Make this a hook and pass `role` to filter out routes.

Reply:
fixed